app/components/user_menu_component.html.erb

```diff
@@ -0,0 +1,52 @@
+<div class="relative mx-4" data-controller='user-menu'>
```

Comment:
Spacing is a little bit different here:

<img width="250" alt="image" src="https://github.com/user-attachments/assets/c7938b15-e5ac-4787-9cf6-8bcdfa71e989" />

from what it was before:

<img width="238" alt="image" src="https://github.com/user-attachments/assets/41909bcd-d18b-49ae-b5a8-473879cc0a9a" />

I think we should be using `mx-3` here which adds `12px` of left and right padding (instead of the `16px` of `mx-4`)